### DISCUSSION TITLE
Zephyrus 2024 G16 4090 power limit

### DISCUSSION THREAD

**Discussion by BravoNorris** [seerge/g-helper#3124] [Q&A] [answered]
Hi, so, some months ago i've followed kind of a tutorial on modernjoshs youtube and modded my G16 2024 with a 175W vbios. He states that he is capable of selecting 175w on GPU power slide and 25w dynamic boost on his g-helper, but i never managed to get that value shown on the sliders. Every g-helper version shows me only 105w and 20w dynamic boost. 

However, i got a really old version from here (0.158.0.0) that allows me to select (and use!) 150w on power slider and 20w dynamic boost.

How can i make the recent versions unlock the 175w +25 power?

In case anyone is wondering, gpu-z shows 175w bios power limit so i'm using the correct vbios.

**Comment by seerge** [comment]
@BravoNorris hello, all recent versions of G-Helper check your VBIOS power limits via 

```nvidia-smi --query-gpu=power.max_limit --format csv,noheader,nounits```

Can you run it in powershell and post output here ?

Then you should see main power slider going up to 150W and boost up to 25W (so 175W total)

**Reply by BravoNorris** [reply]
Hi @seerge , thanks for the reply. I was just trying that as i was digging the discussion posts.

If i run that command, it always shows me:
PS C:\Windows\system32> nvidia-smi --query-gpu=power.max_limit --format csv,noheader,nounits
175.00
doesnt matter what version i'm running, always shows 175.00

If i just run > nvidia-smi, i get:
- 170 when using 0.158.0.0 (if slider is maxed, wich equals 150 + 20)
- 125 when using any other version (105 + 20)

i never got it to show me 25w on max dynamic , always just 20w

My laptop is a GU605MY-QR029W
If you want to take a better look, we can hop on discord and i can share screen with you

**Reply by seerge** [reply]
Ok, interesting. Just in case what language / locale do you use in your windows ?

I see that nvidia-smi outputs value in in US format (i.e. with a dot after 175) and when app reads this number it uses local system number format that can be different from that

**Reply by BravoNorris** [reply]
It's in PT-PT  (portuguese - portugal)

0.158.0.0:
![2024-09-18 00_06_11-Administrator_ Windows PowerShell](https://github.com/user-attachments/assets/9427d35e-2a7e-47b2-8f0b-3fdbe4939f6a)

![2024-09-18 00_09_37-Zephyrus 2024 G16 4090 power limit · seerge_g-helper · Discussion #3124](https://github.com/user-attachments/assets/fcd7ae39-95d4-4ebe-8ec4-d3b37d1c2740)

0.190.0.0:
![2024-09-18 00_11_49-Administrator_ Windows PowerShell](https://github.com/user-attachments/assets/30379279-17c6-4b07-a2cb-15c85c5dcaa5)
![2024-09-18 00_12_01-Zephyrus 2024 G16 4090 power limit · seerge_g-helper · Discussion #3124](https://github.com/user-attachments/assets/94f0dcf4-5f38-4740-b61e-16834a5e6051)

**Reply by seerge** [reply, marked as answer]
@BravoNorris can you try this build ?
[GHelper.zip](https://github.com/user-attachments/files/17040444/GHelper.zip)

**Reply by BravoNorris** [reply]
It works, is this how its supposed to work? 155+20? :) 
<img width="822" alt="ghelper2" src="https://github.com/user-attachments/assets/4e0a4fb0-fac5-4458-b97d-c00e8c7d4dc5">

**Reply by seerge** [reply]
@BravoNorris yeah. As I understand dynamic boost is always 20W in your case ? So it's logical that max base GPU power is just calculated as 175 - 20 = 155 :)

**Reply by BravoNorris** [reply]
Well, i'm not going to complaint!! :) 
Now i can use the updated app with full power.
Hope the solution is universal and can be included in newer updates :)
Thanks a lot @seerge !

**Reply by seerge** [reply]
@BravoNorris great, thanks for spotting. I will include this change in the next release then

**Reply by BravoNorris** [reply]
Thanks for fixing it. Sent you a beer on paypal :) Keep up the awesome work!

**Reply by seerge** [reply]
Thanks
